tests/Annotations/Serializer/HypothesisClientAnnotationNormalizerTest.php

```diff
@@ -179,7 +183,6 @@ public function normalizeProvider() : array
                         'title' => 'title',
                         'uri' => 'uri',
                     ],
-                    'parents' => [],
```

Comment:
to be clear, this is unrelated because the field is optional?

Reply:
Yep.